xml/System.Windows.Forms/TaskDialogPage.xml

```diff
@@ -51,9 +62,22 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets a value that indicates whether the task dialog can be closed with <see cref="P:System.Windows.Forms.TaskDialogButton.Cancel" /> as resulting button by pressing ESC or Alt+F4 or by clicking the title bar's close button, even if a <see cref="P:System.Windows.Forms.TaskDialogButton.Cancel" /> button isn't added to the <see cref="P:System.Windows.Forms.TaskDialogPage.Buttons" /> collection.</summary>
+        <value>
+          <see langword="true" /> to allow to close the dialog by pressing ESC or Alt+F4 or by clicking the title bar's close button; otherwise, <see langword="false" />. The default value is <see langword="false" />.</value>
```

Comment:
Can we put HTML in here? I would use `<kbd>ESC</kbd>` and `<kbd>ALT</kbd>+<kbd>F4</kbd>`

Reply:
I didn't know `<kbd>` were valid tags to add.
Just to make sure I understand, are you asking me if it's possible to add HTML here, or are you just requesting me to add these HTML tags because you know they will work?

Reply:
I'm not seeing any past instances of using `<kbd>` so I'll leave these changes like that.
https://github.com/dotnet/dotnet-api-docs/search?q=extension%3Axml+kbd&unscoped_q=extension%3Axml+kbd